packages/web-app-admin-settings/src/components/Spaces/ContextActions.vue

```diff
@@ -34,6 +36,8 @@ export default defineComponent({
     }
   },
   setup(props) {
+    const store = useStore()
+    const maxQuota = computed(() => store.getters.capabilities.spaces?.max_quota)
```

Comment:
Please introduce a new composable in `useCapability.ts` for this (see examples there).